script/Arcueid/Day 5/05_00_ARC05_3.txt

```diff
@@ -6,17 +6,17 @@
 [sha:bcac3d7504b37322c7785e42c0b22a5b7f737149]{
 -- Page 10005, Offset 9133.
 -- 志貴が一緒に戦ってくれるのは助かるしね。あなたならヴローヴなんて問題なく一撃で“殺”せるんだから」
-I'm glad that you've decided to fight by my side. After all, we can easily 'kill' Vlov in one blow with you around."
+I'm glad that you've decided to fight by my side. After all, with you around, we'll definitely be able to 'kill' Vlov with a single strike."
 }
 [sha:76a2ccc7a265ad4a64465ceb5c9dce093ec2dc1c]{
 -- Page 218, Offset 9134.
 -- 「……いや、それは無理だろ？
-"...No, surely that's not possible.
+"...No, I doubt that'll happen.
```

Comment:
Kinda gives slightly the wrong impression imo, this makes it sound like it won't happen _that way_ rather than "that's impossible lmao"
> ...No, I can't see how that would work.